src/View/Admin/WooTab.php

```diff
@@ -891,6 +891,6 @@ private function get_connection_button() : array {
 	 * @return bool
 	 */
 	private function has_active_settings_section() : bool {
-		return ! empty( $GLOBALS['current_section'] );
+		return ! empty( $GLOBALS['current_section'] ?? '' );
 	}
```

Comment:
Not sure if this one is needed, unless you were by chance seeing a notice for it on your own. I think the empty() check covers the notices though.

Reply:
@tw2113 the `empty()` check only checks the value of `$GLOBALS['current_section']`, so my understanding is that it could still trigger the "undefined index" warning if the key doesn't exist at all. To actually avoid the warning, we'd need to either do an `isset()` check first, or just use the null-coalescing operator.